src/main/java/com/legobmw99/allomancy/modules/powers/handlers/CommonEventHandler.java

```diff
@@ -336,23 +338,38 @@ public void onWorldTick(final TickEvent.WorldTickEvent event) {
                         curPlayer.removePotionEffect(Effects.NIGHT_VISION);
                     }
                     if (cap.isBurning(Metal.PEWTER)) {
+                        FoodStats foodStats = curPlayer.getFoodStats();
+                        if (foodStats.getFoodLevel() < 20) {
+                            cap.setHungerStored(cap.getHungerStored() + 20 - foodStats.getFoodLevel());
+                            foodStats.setFoodLevel(20);
+                        }
```

Comment:
I'd be interested in a discussion of more of this idea. It's interesting in the lore viewpoint of increased endurance, but I'm not super happy with the way health is done currently in the mod so I don't want to replicate it further. It also seems like there is no way to reduce this effect while burning pewter? Stored health is slowly reduced as you burn pewter for longer, but this provides no way to eat while burning pewter which seems like it would be frustrating

Reply:
I agree, maybe pewter could set the player's health just above sprinting level, it wouldn't look pretty, but it would let you eat and achieve the desired effect.

Reply:
I think I'm going to wait and see what 1.16 does with the proposed saturation and hunger changes before touching this